example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.6.0'
+    ext.kotlin_version = '1.7.10'
```

Comment:
This is just the example, right? Won't really affect anything in our SDK

Reply:
Yes.

Reply:
OK. I guess we are not bumping up version after this change? More like to have another PR?